openedx/core/djangoapps/theming/models.py

```diff
@@ -17,3 +18,21 @@ class SiteTheme(models.Model):
 
     def __unicode__(self):
         return self.theme_dir_name
+
+    @staticmethod
+    def get_theme(site):
+        """
+        Get SiteTheme object for given site, returns default site theme if it can not
+        find a theme for the given site and `DEFAULT_SITE_THEME` setting has a proper value.
+
+        Args:
+            site (django.contrib.sites.models.Site): site object related to the current site.
+
+        Returns:
+            SiteTheme object for given site or a default site set by `DEFAULT_SITE_THEME`
+        """
+
+        theme = site.themes.first()
```

Comment:
@saleem-latif Makes sense. It would be great if possible if we could get it in this week so that we can use it to diagnose any issues that come up during the release. It seems like it should be a very simple change.